prime-router/docs/releases/2021-05-27-release-notes.md

```diff
@@ -0,0 +1,47 @@
+#  Hub Release Notes
+
+*May 27, 2021*
+
+## General useful links:
+
+- All Schemas are documented here:  [Link to detailed schema dictionaries](../schema_documentation)
+- The Hub API is documented here: [Hub OpenApi Spec](../openapi.yml)
+- [Click here for all release notes](../releases)
+
+## For this release
+
+### Addition of Quality Check
+
+A basic Quality check on certain fields is now applied to Covid-19 data before sending it to downstream public health departments.
```

Comment:
This is a repeat of what was in the last Release Notes, but this is much more readable, and includes the latest tweak, to include CLIA.